Binance.Net/Clients/CoinFuturesApi/BinanceRestClientCoinFuturesApiAgent.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Binance.Net.Enums;
+using Binance.Net.Interfaces.Clients.CoinFuturesApi;
+using Binance.Net.Objects.Models.Futures;
+
+namespace Binance.Net.Clients.CoinFuturesApi
+{
+    /// <inheritdoc />
+    internal class BinanceRestClientCoinFuturesApiAgent : IBinanceRestClientCoinFuturesApiAgent
+    {
+        private static readonly RequestDefinitionCache _definitions = new RequestDefinitionCache();
+
+        private readonly BinanceRestClientCoinFuturesApi _baseClient;
+
+        internal BinanceRestClientCoinFuturesApiAgent(BinanceRestClientCoinFuturesApi baseClient)
+        {
+            _baseClient = baseClient;
+        }
+
+        #region If New User
+
+        public async Task<WebCallResult<BinanceIfNewUser>> IfNewUser(string brokerId, int? receiveWindow = null, CancellationToken ct = default)
+        {
+            brokerId.ValidateNotNull(nameof(brokerId));
+
+            var parameters = new ParameterCollection();
+            parameters.AddOptional("brokerId", brokerId);
```

Comment:
You removed the `ToString`, but it still should be `parameters.Add` instead of `parameters.AddOptional`